tasks/ModelTask.cpp

```diff
@@ -30,101 +29,102 @@ void ModelTask::setGazeboModel(WorldPtr _world,  ModelPtr _model)
 
 	world = _world;
 	model = _model;
-//	sdf = model->GetSDF();
 	
-//	Export Gazebo Joints and Links to Rock-Robotics
 	setJointPorts();
-	setLinkPorts();
 } 
-//======================================================================================
+
 void ModelTask::setJointPorts()
```

Comment:
setJointPorts really does not describe what the method does. Better name: setupJoints